docs/docs/unit-testing.md

```diff
@@ -249,9 +246,15 @@ module.exports = {
 }
 ```
 
-You may notice that two other options, `testRegex` and `moduleFileExtensions`,
-have been added. Option `testRegex` is the pattern telling Jest which files
-contain tests. The pattern above matches any `.js`, `.jsx`, `.ts` or `.tsx`
+Also update `jest.preprocess.js` with the following babel preset to look like this:
+
+```js:title=jest-preprocess.js
+const babelOptions = {
+  presets: ["babel-preset-gatsby", "@babel/preset-typescript"],
+}
+```
+
+The pattern above matches any `.js`, `.jsx`, `.ts` or `.tsx`
```

Comment:
We don't have "pattern above" anymore, so if we chose to keep information what tests will be picked up we should mention that `jest` by default will use those instead and link to `jest` config docs

Reply:
I went ahead and did a bigger cleanup and removed more redundant references.